CTP/shell/src/com/navercorp/cubridqa/shell/common/CommonUtils.java

```diff
@@ -280,6 +283,25 @@ public static String toFixedLength(String str, int len, char fillChar) {
 		return result.substring(result.length() - len);
 	}
 	
+	public static void generateFailBackupPackage(Context context){
+		if(context.isWindows()) return;
```

Comment:
as is:
if(context.isWindows()) return;

The context.isWindows() always means the test node is in windows platform. It should not be used to controller platform.